presentation/common/src/presentation-common/content/Descriptor.ts

```diff
@@ -432,3 +388,22 @@ export class Descriptor implements DescriptorSource {
     return overrides;
   }
 }
+
+/** @alpha */
+export namespace Descriptor {
```

Comment:
`Descriptor` is a class, so there's no need to add the function to a namespace - you can just add it to `Descriptor` class a a `static` function